src/main/java/frc/robot/commands/Auto/ScoreMidDriveBack.java

```diff
@@ -34,17 +34,13 @@ public ScoreMidDriveBack(DriveSubsystem m_drive, FieldSim m_fieldSim, ArmInOut m
 
     // adds commands to run sequentially when executed
     addCommands(
-        // run instantly
-        new InstantCommand(m_LED::autoStart),
 
         new ManipulatorMidGoal(m_ArmPivot, m_ArmInOut, m_Wrist, m_LED).withTimeout(1),
         new RunCommand(m_Gripper::GripperOut, m_Gripper).withTimeout(2),
         new RunCommand(m_Gripper::GripperStop, m_Gripper).withTimeout(0.1),
-        // new FollowTrajectory(m_drive, m_fieldSim, "Drive Backward", m_ArmInOut,
-        // m_ArmPivot, m_Gripper, m_Wrist, m_LED),
         new ManipulatorGroundAuto(m_ArmPivot, m_ArmInOut, m_Wrist, m_LED).withTimeout(0.1),
-        new RunCommand(() -> m_drive.drive(0.3, 0, 0, true, true), m_drive).withTimeout(2),
-        // new ManipulatorZero(m_ArmPivot, m_ArmInOut, m_Wrist, m_LED).withTimeout(2),
-        new InstantCommand(m_LED::autoEnd));
+        new RunCommand(() -> m_drive.drive(0.3, 0, 0, true, true), m_drive).withTimeout(2)
```

Comment:
Unrelated to your changes, but I don't think we ever tested that this still goes the distance we'd expect with the extra weight added since NDGF. If we get practice field time or if we run one of the drive back autos on Thursday, this should definitely get adjusted if needed

Reply:
That should be tested